tests/flags/flags.go

```diff
@@ -73,6 +75,8 @@ func init() {
 	flag.StringVar(&PathToTestingInfrastrucureManifests, "path-to-testing-infra-manifests", "manifests/testing", "Set path to testing infrastructure manifests")
 	flag.StringVar(&PreviousReleaseTag, "previous-release-tag", "", "Set tag of the release to test updating from")
 	flag.StringVar(&PreviousReleaseRegistry, "previous-release-registry", "quay.io/kubevirt", "Set registry of the release to test updating from")
+	flag.StringVar(&PreviousUtilityRegistry, "previous-utility-container-registry", KubeVirtRepoPrefix, "Set registry of the utility containers to test updating from")
```

Comment:
I think, while setting a default value here works in our CI, it may introduce another moving variable which needs to be set if `quay.io/kubevirt` is change for `previous-relese-registry`.

Would it be possible for you to sort out setting defaults after the flag parsing? On initial thought from my side it seems to make sense, that if I don't specify a utility repo that it will automatically fall back to `PreviousReleaseRegistry`.

Does that make sense?

Reply:
@rmohr Thanks for the review. If I understand correctly, you are suggesting that when `PreviousUtilityRegistry`  is not given from command line, it's better to be set to `PreviousReleaseRegistry` as a fallback, right?  If yes, I agree. How about putting this in [here](https://github.com/kubevirt/kubevirt/blob/1ca2297bb308787ccd5b8e1fa728b0a201c83df5/tests/flags/flags.go#L93)?

Reply:
You understood it correct and that is the perfect place :+1:

Reply:
Good! Thanks for confirmation. Done.